ceph-salt-formula/salt/ceph-salt/common/sshkey.sls

```diff
@@ -2,63 +2,63 @@
 
 {{ macros.begin_stage('Distribute SSH keys') }}
 
-{% set ssh_user = pillar['ceph-salt']['ssh']['user'] %}
-{% set ssh_user_group = 'root' if ssh_user == 'root' else 'users' %}
-
-{% if ssh_user != 'root' %}
-
 create ssh user group:
   group.present:
-    - name: {{ ssh_user_group }}
+    - name: users
 
 create ssh user:
   user.present:
-    - name: {{ ssh_user }}
-    - home: /home/{{ssh_user}}
+    - name: cephadm
+    - home: /home/cephadm
     - groups:
-      - {{ ssh_user_group }}
+      - users
     - failhard: True
 
-{% endif %}
-
-{{ macros.sudoers('configure sudoers') }}
+configure sudoers:
+  file.append:
+    - name: /etc/sudoers.d/ceph-salt
+    - text:
+      - "cephadm ALL=NOPASSWD: /usr/bin/ceph -s"
+      - "cephadm ALL=NOPASSWD: /usr/bin/ceph orch host add *"
+      - "cephadm ALL=NOPASSWD: /usr/bin/ceph orch host ok-to-stop *"
+      - "cephadm ALL=NOPASSWD: /usr/bin/ceph orch status --format=json"
```

Comment:
instead of taxatively listing specific `ceph` CLI commands, would it make sense to replace all of these lines with:

```
"cephadm ALL=NOPASSWD: /usr/bin/ceph *"
```

Reply:
I can't help thinking that users are going to want to log in as the `cephadm` user on the Salt Master node, and use it to administer the Ceph cluster (instead of being logged in as `root` to do that).